docs/shared/contribution_guidelines/metadata_schemas.rst

```diff
@@ -44,3 +44,26 @@ Development guidelines
 #. Within a schema, only properties that can be expected for all respected real-world entities should be made required.
 #. If properties of a real-world entity contains potentially sensitive information (e.g., personal data), these properties should be specified in a separate schema.
 #. Schema properties should be consistently reduced to a minimum, meaning the reuse of a property name across schemas is highly recommended, if the definition of that property name remains the same.
+
+Current schemas
```

Comment:
These are not schemas... these are metadata (sub)models consisting of multiple schemas.
The metadata models should have their own contribution guidelines.

Reply:
I would probably link from here to the contributions for metadata models. In principle it would be smart to autogenerate the metadata models page so that it is always reflecting the latest state of extensions.

Reply:
@apdavison thank you for helping pushing / completing the documentation. this is highly appreciated :smile:

Reply:
@apdavison based on our discussion I will delete that part and replace it with a general linkage to the main GitHub for issues on schemas (or extensions)